app/src/main/kotlin/com/wire/android/ui/home/conversationslist/ConversationListViewModel.kt

```diff
@@ -163,4 +167,14 @@ class ConversationListViewModel @Inject constructor(
             }
         }
     }
+
+    private fun mapUserType(userType: UserType): Membership {
+        return when (userType) {
+            UserType.GUEST -> Membership.Guest
+            UserType.FEDERATED -> Membership.Federated
+            UserType.EXTERNAL -> Membership.External
+            UserType.INTERNAL -> Membership.None
+            else -> {  throw IllegalStateException("Unknown UserType") }
```

Comment:
Why this else?

IDE might complain a bit, but the compiler knows that this `when` is exhaustive and there's no need for it.